### PR TITLE
Update API for OIDC scopes

### PR DESCRIPTION
**Pull request by dwendland**

### CODE REVIEW

File: src/main/java/org/fiware/iam/ServiceMapper.java [hunk 1]
```diff
@@ -1,103 +1,130 @@
...
+        }
+        Credential credential = new Credential()
+                .setCredentialType(credentialVO.getType());
+        List<EndpointEntry> trustedList = new ArrayList<>();
+        Optional.ofNullable(issuersToEntries(credentialVO.getTrustedIssuersLists())).ifPresent(trustedList::addAll);
+        Optional.ofNullable(participantsToEntries(credentialVO.getTrustedParticipantsLists())).ifPresent(trustedList::addAll);
+        credential.setTrustedLists(trustedList);
+        return credential;
+    }
+
```
Comment (wistefan) [inline src/main/java/org/fiware/iam/ServiceMapper.java:56]: isn't that a mapping that mapstruct can construct?

**Review by wistefan — APPROVED** [review]

File: src/main/resources/db/migration/V1_0_2__minimal_table.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+DROP TABLE `service` CASCADE;
```
Comment (beknazaresenbek) [inline src/main/resources/db/migration/V1_0_2__minimal_table.sql:1]: I think, here it'll fail, if the table doesn't exist. 'IF EXISTS' would prevent an error

Comment (pulledtim) [reply src/main/resources/db/migration/V1_0_2__minimal_table.sql:1]: The table is created in the previous db migration step, so i deemed it not necessary :sweat_smile:

Comment (wistefan) [reply src/main/resources/db/migration/V1_0_2__minimal_table.sql:1]: sleepy-sloth

File: src/main/java/org/fiware/iam/ServiceMapper.java (hunk 1 above)
Comment (pulledtim) [reply src/main/java/org/fiware/iam/ServiceMapper.java:56]: Hem for the trustedLists field, two other fields are mapped and then merged into one. At least I didn't try something like that yet

**Pull request merged**